trpcultivate_germplasm/tests/src/Kernel/TripalImporter/GermplamCollectionImporter/GermplasmCollectionImporterRunTest.php

```diff
@@ -0,0 +1,517 @@
+   *
+   * @var array
+   */
+  protected array $definitions = [
+    'test-collection-importer' => [
+      'id' => 'trpcultivate-germplasm-population-importer',
+      'label' => 'Tripal Importer: Germplasm Collection Importer',
+      'description' => 'Imports germplasm populations (i.e. RIL, NAM, cross progeny) into testchado.',
+      'file_types' => ['tsv', 'txt'],
+      'upload_title' => 'Population Individuals*',
+      'upload_description' => 'This should not be visible!',
+      'use_analysis' => FALSE,
+      'require_analysis' => FALSE,
+      'use_button' => TRUE,
+      'submit_disabled' => FALSE,
+      'button_text' => 'Import',
+      'file_upload' => TRUE,
+      'file_local' => FALSE,
+      'file_remote' => FALSE,
+      'file_required' => TRUE,
+      'cardinality' => 1,
+      'menu_path' => '',
+      'callback' => '',
+      'callback_module' => '',
+      'callback_path' => '',
+    ],
+  ];
+
+  /**
+   * The path to tripalcultivate_germplasm module.
+   *
+   * @var string
+   */
+  private $module_path;
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Ensure we see all logging in tests.
+    \Drupal::state()->set('is_a_test_environment', TRUE);
+
+    // Open connection to Chado.
+    $this->chado_connection = $this->getTestSchema(ChadoTestKernelBase::PREPARE_TEST_CHADO);
+
+    // Ensure we can access file_managed related functionality from Drupal.
+    // ... users need access to system.action config?
+    $this->installConfig(['system', 'trpcultivate_germplasm', 'trpcultivate']);
+    // ... managed files are associated with a user.
+    $this->installEntitySchema('user');
+    // ... Finally the file module + tables itself.
+    $this->installEntitySchema('file');
+    $this->installSchema('file', ['file_usage']);
+    $this->installSchema('tripal_chado', ['tripal_custom_tables']);
+    // Ensure we have our tripal import tables.
+    $this->installSchema('tripal', ['tripal_import', 'tripal_jobs']);
+    // Create and log-in a user.
+    $this->setUpCurrentUser();
+
+    // We need to mock the logger to test the progress reporting.
+    $container = \Drupal::getContainer();
+    $mock_logger = $this->getMockBuilder(TripalLogger::class)
+      ->onlyMethods(['error'])
+      ->getMock();
+
+    $mock_logger->method('error')
+      ->willReturnCallback(function ($message, $context, $options) {
+        return NULL;
+      });
+    $container->set('tripal.logger', $mock_logger);
+
+    // Create our organism and configure it.
+    $organism_id = $this->chado_connection->insert('1:organism')
+      ->fields([
+        'genus' => 'Lens',
+        'species' => 'culinaris',
+      ])
+      ->execute();
+    $this->assertIsNumeric($organism_id,
+      "We were not able to create an organism for testing.");
+    $this->chado_connection->insert('1:organism')
+    ->fields([
+      'genus' => 'Lens',
+      'species' => 'ervoides',
+    ])
+    ->execute();
```

Comment:
```suggestion
      ->fields([
        'genus' => 'Lens',
        'species' => 'ervoides',
      ])
      ->execute();
```

Line has line indented incorrectly error.